resource.go

```diff
@@ -28,6 +31,11 @@ type Resource interface {
 
 	UID() int64
 	GID() int64
+
+	ATime() time.Time
+	MTime() time.Time
+	CTime() time.Time
+	BTime() time.Time
```

Comment:
I think it's fine to use these abbreviated forms - they're pretty widely used. But it would be useful to have a comment explaining what each one means.

Reply:
Document these. Please change to the expanded versions to match golang os package.